lib/virt_disk/block_file.rb

```diff
@@ -1,9 +1,10 @@
 module VirtDisk
   class BlockFile
-    attr_accessor :path
+    attr_accessor :path, :offset
 
-    def initialize(path)
+    def initialize(path, offset=0)
```

Comment:
@roliveri right so until we have that partition logic any images / filesystems we generate to anaylze w/ virtfs will require some manually set partition offset.

We could do a 'dd' after generating each one to get rid of the leading partition metadata and shift the beginning of the filesystem to the beginning of the file, but this would allow us to set the offset for subsequent lookup here without having to do that.

In the virtfs-refs test script I use the known refs partition offset when mounting the filesystem. All calls to retrieve fs block data are relative to that:

https://github.com/movitto/virtfs-refs/blob/master/examples/test.rb

Again, this is just a workaround, when we have the partition logic & supporting framework, we can remove this offset bit.